src/commands/update.cr

```diff
@@ -18,6 +23,10 @@ module Shards
 
         manager.to_lock(lockfile_path)
       end
+
+      private def has_dependencies?
+        spec.dependencies.any? || !shards.production? || spec.development_dependencies.any?
```

Comment:
Ahh, I lifted it from: https://github.com/crystal-lang/shards/blob/master/src/commands/check.cr#L20